.github/workflows/check-for-update.yaml

```diff
@@ -17,8 +17,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: use node 14
-        uses: actions/setup-node@v3
+      - uses: actions/setup-node@v3
```

Comment:
version being used still explicit in the logs?

Reply:
Only if you open the section. But not needed and can drift.
For tests where we explicitly want to test the node version we use a matrix and it's visible